finagle/h2/src/main/scala/com/twitter/finagle/buoyant/h2/H2Transport.scala

```diff
@@ -37,5 +37,7 @@ object H2Transport {
     def goAway(err: GoAway, deadline: Time): Future[Unit]
 
     final def goAway(err: GoAway): Future[Unit] = goAway(err, Time.Top)
+
+    def sendPing(): Future[Unit]
```

Comment:
:+1: to calling this `sendPing`.  I still think a comment on the semantics wouldn't hurt.